kolibri/core/device/api.py

```diff
@@ -113,6 +113,22 @@ def get(self, request, format=None):
         info["python_version"] = "{major}.{minor}.{micro}".format(
             major=version_info.major, minor=version_info.minor, micro=version_info.micro
         )
+
+        if not request.user.is_superuser:
+            # If user is not superuser, return just free space available and kolibri version
+            keys_to_remove = [
+                "urls",
+                "database_path",
+                "device_id",
+                "os",
+                "server_time",
+                "server_timezone",
+                "installer",
+                "python_version",
+            ]
+            for key in keys_to_remove:
+                info.pop(key)
```

Comment:
I know I'm being nitpicking but if you are not going to use the retrieved value, `del` is recommended instead of `pop`

Reply:
Not nitpicking - helpful for me to know! I'll fix.

Reply:
Fixed